### DISCUSSION TITLE
Square for putting, Mevo+ for the other shots?

### DISCUSSION THREAD

**Discussion by kahis551** [springbok/MLM2PRO-GSPro-Connector#144] [Ideas]
Would it be possible to develop the connector so that it could enable putting with the Square and the other shots with another LM?

I have a Mevo+ for my practice needs and am considering also getting the Square, both for the short game capabilities and the casual play (swing stick) for family entertainment. It would be great to be able to combine the strengths of them.

If I understand correctly, the GSPro Cloud already provides this by allowing the user to use two launch monitors?

**Comment by wonder99** [comment]
This seems doable--instead of looking for an exputt screen, it would look
at the Square screen.
I'm not very familiar with GSPro Cloud, but I'm not sure how it uses 2
LMs.  But this connector can just do OCR on two screens, one for putts and
another for the rest.
So while it seems very doable to putt with square, full shots with mevo+,
it would not be doable to use square for both putting and short chips.
- Paul

**Reply by kahis551** [reply]
That would be ok for me, since the Mevo+ works fine for even short chips in my setup. The putting is the area the radar is not good in.
